scrapy_poet/injection.py

```diff
@@ -61,6 +66,17 @@ def load_providers(self, default_providers: Optional[Mapping] = None):
         self.is_class_provided_by_any_provider = \
             is_class_provided_by_any_provider_fn(self.providers)
 
+    def init_cache(self):
+        self.cache = None
+        cache_filename = self.spider.settings.get('SCRAPY_POET_CACHE')
+        if cache_filename and isinstance(cache_filename, bool):
+            cache_filename = os.path.join(get_scrapy_data_path(createdir=True), "scrapy-poet-cache.sqlite3")
+        if cache_filename:
+            compressed = self.spider.settings.getbool('SCRAPY_POET_CACHE_GZIP', True)
+            self.caching_errors = self.spider.settings.getbool('SCRAPY_POET_CACHE_ALSO_ERRORS', False)
```

Comment:
What do you think about calling the option just `SCRAPY_POET_CACHE_ERRORS`?

Reply:
+1 on this. Updated on https://github.com/scrapinghub/scrapy-poet/pull/55/commits/df29c938bf888b13339c52003b2eaa5e16a8a16d